public/app/core/components/Breadcrumbs/BreadcrumbItem.tsx

```diff
@@ -74,6 +74,7 @@ const getStyles = (theme: GrafanaTheme2) => {
       flex: 1,
       minWidth: 0,
       maxWidth: 'max-content',
+      padding: theme.spacing(0.5, 0.5),
```

Comment:
Does it still work okay with 0 padding horizontally?

Reply:
Without horizontal padding, the first item of the breadcrumbs does not have the left blue border when on focus.
<img width="354" alt="Captura de pantalla 2024-09-09 a las 16 41 51" src="https://github.com/user-attachments/assets/36c910fa-8478-46b8-a4a2-a7857180c321">

Reply:
if we keep this padding but also remove the padding on line 60 then i think the breadcrumb placement remains unchanged 👍 can we try that?